core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,12 +436,23 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
-              ExplorationRecommendationsService.getRecommendedSummaryDicts(
-                ExplorationPlayerStateService.getAuthorRecommendedExpIds(
-                  stateName),
-                function(summaries) {
-                  $scope.recommendedExplorationSummaries = summaries;
-                });
+              var parentExplorationId = UrlService.getParentExplorationId();
+              if (parentExplorationId) {
+                $scope.parameters = UrlService.getParameters();
+                ExplorationRecommendationsService.getRecommendedSummaryDicts(
+                  [parentExplorationId],
+                  function(summaries) {
+                    $scope.recommendedExplorationSummaries = summaries;
+                  });
+              } else {
+                $scope.parameters = null;
```

Comment:
Set $scope.parentExplorationIds to null at the outset, don't bury it here. Null should be the default value.

Reply:
Done, changed the format of variable here